test/README.md

```diff
@@ -0,0 +1,48 @@
+# CI/CD for infrastructure deployment & synapse pipelines
+
+These tests would deploy the Azure infrastructure using bicep templates and runs the synapse pipeline to perform test on the checked-in code.
+
+We are also making use of a pre-provisioned batch account and we deploy a batch-account pool and job on batch-account to override some quota challenges faced in our Azure subscription.
+
+## Github secrets and credentials
+
+We make use of github secrets to provide credentials and other secret information to run the pipeline.
+
+Following is the list of secrets we are currently using:
+
+- AZURE_CREDENTIALS:
+    These are the credentials used accessing azure cloud. Follow the instructions on the [documentation](https://docs.microsoft.com/en-us/azure/developer/github/connect-from-azure?tabs=azure-cli%2Clinux#use-the-azure-login-action-with-a-service-principal-secret) to create and use the credentials.
+
+- CI_BATCH_ACCOUNT_NAME:
+    This is the pre-provisioned batch account name.
```

Comment:
What does this mean for external users? Do they need to provision the batch account first?

Reply:
As of now yes. We would update the test script in the next iteration that if this secret is not provided then we would create the batch account